### ISSUE TITLE
Android: Failed resolution of: Landroidx (Didn't find class "androidx.localbroadcastmanager.content.LocalBroadcastManager")

### ISSUE THREAD

**Issue by newfylox** [react-native-kit/react-native-track-player#1132]
I just upgraded from 1.2.3 to 1.2.4 and got this error. I didn't make any changes to my project.

I tried a `react-native link` with a `clean` and `rebuild project` in AndroidStudio without any success.

Any clue? Thanks!

```
    java.lang.NoClassDefFoundError: Failed resolution of: Landroidx/localbroadcastmanager/content/LocalBroadcastManager;
        at com.guichaguri.trackplayer.module.MusicModule.initialize(MusicModule.java:49)
        at com.facebook.react.bridge.ModuleHolder.doInitialize(ModuleHolder.java:223)
        at com.facebook.react.bridge.ModuleHolder.markInitializable(ModuleHolder.java:98)
        at com.facebook.react.bridge.NativeModuleRegistry.notifyJSInstanceInitialized(NativeModuleRegistry.java:102)
        at com.facebook.react.bridge.CatalystInstanceImpl$2.run(CatalystInstanceImpl.java:407)
        at android.os.Handler.handleCallback(Handler.java:938)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:29)
        at android.os.Looper.loop(Looper.java:223)
        at com.facebook.react.bridge.queue.MessageQueueThreadImpl$4.run(MessageQueueThreadImpl.java:232)
        at java.lang.Thread.run(Thread.java:923)
     Caused by: java.lang.ClassNotFoundException: Didn't find class "androidx.localbroadcastmanager.content.LocalBroadcastManager" on path: DexPathList[[zip file "/data/app/~~5uK9Y-D-ChIvWQfauCmbgA==/com.myapp-9d0hU3xmKpsnHaBmCiYgiQ==/base.apk"],nativeLibraryDirectories=[/data/app/~~5uK9Y-D-ChIvWQfauCmbgA==/com.myapp-9d0hU3xmKpsnHaBmCiYgiQ==/lib/arm64, /data/app/~~5uK9Y-D-ChIvWQfauCmbgA==/com.myapp-9d0hU3xmKpsnHaBmCiYgiQ==/base.apk!/lib/arm64-v8a, /system/lib64, /system_ext/lib64]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:207)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        at com.guichaguri.trackplayer.module.MusicModule.initialize(MusicModule.java:49) 
        at com.facebook.react.bridge.ModuleHolder.doInitialize(ModuleHolder.java:223) 
        at com.facebook.react.bridge.ModuleHolder.markInitializable(ModuleHolder.java:98) 
        at com.facebook.react.bridge.NativeModuleRegistry.notifyJSInstanceInitialized(NativeModuleRegistry.java:102) 
        at com.facebook.react.bridge.CatalystInstanceImpl$2.run(CatalystInstanceImpl.java:407) 
        at android.os.Handler.handleCallback(Handler.java:938) 
        at android.os.Handler.dispatchMessage(Handler.java:99) 
        at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:29) 
        at android.os.Looper.loop(Looper.java:223) 
        at com.facebook.react.bridge.queue.MessageQueueThreadImpl$4.run(MessageQueueThreadImpl.java:232) 
        at java.lang.Thread.run(Thread.java:923) 
```

**Comment by newfylox** [comment]
It seems that adding `implementation 'androidx.localbroadcastmanager:localbroadcastmanager:1.0.0'` in my `app/build.gradle` dependencies fixed it, right after `implementation "com.facebook.react:react-native:+"`.

Should it be a part of the package dependencies instead, since automatic linking exists in React-Native?

**Comment by curiousdustin** [comment]
@newfylox, thank you for reporting this. I am able to reproduce this issue and I am taking a look.

**Comment by curiousdustin** [comment]
@newfylox, can you test out this change, by installing from this specific commit?
``` json
"react-native-track-player": "git@github.com:react-native-kit/react-native-track-player.git#bdd731f6dd76e14431d3a97b2950ec0f9ac8dc1d"
```

**Comment by newfylox** [comment]
@curiousdustin I had to update the line for `package.json` like this

```json
"react-native-track-player": "https://github.com/react-native-kit/react-native-track-player.git#bdd731f6dd76e14431d3a97b2950ec0f9ac8dc1d",
```

Then I cleaned and rebuild the project and it worked like a charm!

**Comment by curiousdustin** [comment]
Ok, thanks, we will get this released shortly.

**Comment by curiousdustin** [comment]
Fix released in [1.2.5](https://github.com/react-native-kit/react-native-track-player/releases/tag/v1.2.5).

**Comment by Sateesh7070** [comment]
https://github.com/mauron85/react-native-background-geolocation/issues/584
can you any one please help me on this.
